example_sp/README.md

```diff
@@ -0,0 +1,25 @@
+## Start Serice Provider ##
+
+Syntax: ./start.sh -u URL -p PATH -c
+
+* -u URL is the idp metadata url, is mandatory the presence of an url or the metadata/idp.xml file 
+* -p PATH if present, copy the sp metadata on path
+* -c clean log, certificates and metadata at end of script
+
+### Example ###
+```
+$ ./start.sh -u https://idp.example.org/metadata -p /opt/satosa/metadata/sp
+```
+make new certificates, build a conf with idp metadata from idp.example.org and copy the sp metadata on satosa path
```

Comment:
Consider this example, that's good enough :)
https://uniauth.readthedocs.io/contents/installation.html#create-your-own-saml-certificates

Reply:
Fixed ab8e23053ecc5fa8194d29ae18a135a3cf86125e

Reply:
Ignore last message.....
ok, now I have used this function for key generation:
```
openssl req -nodes -new -x509 -newkey rsa:2048 -days 3650 -keyout pki/mykey.pem -out pki/mycert.pem -subj "/C=IT/ST=Rome/L=Rome/O=test/OU=test/CN=localhost"
```